src/core_io.h

```diff
@@ -39,7 +39,7 @@ std::vector<unsigned char> ParseHexUV(const UniValue& v, const std::string& strN
 int ParseSighashString(const UniValue& sighash);
 
 // core_write.cpp
-UniValue ValueFromAmount(const CAmount& amount);
+UniValue ValueFromAmount(const CAmount amount);
```

Comment:
Thanks! Can you please drop the `const` here and in FormatMoney (discussion why: https://github.com/bitcoin/bitcoin/pull/19845#discussion_r489776345)

Reply:
`void f(const int i)` vs `void f(int i)` is a matter of taste :)  Leaving as-is.

Reply:
IMO `const` should be used for the definition, but not declaration, when appropriate. In the definition, it instructs the compiler to error if you attempt to change it (eg, a bug). In the declaration, it has no effect at all.

Reply:
FWIW the original idea with passing a reference here is that if CAmount is a more complex object, a copy is not needed. At the time, this helped e.g. sidechains and altcoins with different concept of amount (say, a variable length integer) easier use the code. I don't care much about this, but just thought I'd mention.

I agree with @luke-jr that const in by-value definitions doesn't do anything and so is just clutter.

Reply:
Seems unrelated to a bugfix in any case